sdcm/cluster.py

```diff
@@ -872,6 +879,12 @@ def node_setup(node):
     def run_stress_thread(self, stress_cmd, timeout, output_dir):
         queue = Queue.Queue()
 
+        test_upgrade = self.params.get('test_upgrade')
+        if test_upgrade == "True":
```

Comment:
It seems you forgot to add the `test_upgrade` param to the YAML file. If it's not there, people will forget to set it.